tests/integration/ha/test_small_deployment_upgrades.py

```diff
@@ -81,69 +83,43 @@ async def test_deploy_latest_from_channel(ops_test: OpsTest) -> None:
     )
     assert len(ops_test.model.applications[APP_NAME].units) == 3
 
+    await set_watermark(ops_test, APP_NAME)
 
-@pytest.mark.runner(["self-hosted", "linux", "X64", "jammy", "xlarge"])
-@pytest.mark.group(1)
-@pytest.mark.abort_on_fail
-async def test_upgrade_rollback(
-    ops_test: OpsTest, c_writes: ContinuousWrites, c_writes_runner
-) -> None:
-    """Test upgrade from upstream to currently locally built version."""
+
+@pytest.fixture()
+async def c_writes(ops_test: OpsTest):
```

Comment:
This is already part of the `conftest.py` - there is no need to define it again.